config/routes.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
   resources :posts
   devise_for :users
-  root "posts#index"
+  root 'posts#index'
```

Comment:
- [ ] It creates the application routes using `resources :posts, only: [:new, :create, :index]` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/posts_routes_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/authentication)). Kindly implement this.